### ISSUE TITLE
Claiming OSDs after a backend has been linked to a global backend is not possible

### ISSUE THREAD

**Issue by JeffreyDevloo** [openvstorage/framework-alba-plugin#554]
### Problem description
After linking the local backends to global ones, a OSD entry is made without IPs.
The add_generic_osds function loops over all OSDs, including the backend entry and then loops over ips: resulting in an error

Stacktrace:
```
Jan 26 12:05:32 DEV-4N-199-191 celery[610]: 2018-01-26 12:05:32 26500 +0100 - DEV-4N-199-191 - 610/140365147571968 - celery/log.py - log - 8468 - ERROR - Task alba.add_osds[6a3514c3-8897-461f-8839-78e35022db6a] raised unexpected: T
Jan 26 12:05:32 DEV-4N-199-191 celery[610]: Traceback (most recent call last):
Jan 26 12:05:32 DEV-4N-199-191 celery[610]:   File "/usr/lib/python2.7/dist-packages/celery/app/trace.py", line 240, in trace_task
Jan 26 12:05:32 DEV-4N-199-191 celery[610]:     R = retval = fun(*args, **kwargs)
Jan 26 12:05:32 DEV-4N-199-191 celery[610]:   File "/usr/lib/python2.7/dist-packages/celery/app/trace.py", line 438, in __protected_call__
Jan 26 12:05:32 DEV-4N-199-191 celery[610]:     return self.run(*args, **kwargs)
Jan 26 12:05:32 DEV-4N-199-191 celery[610]:   File "/opt/OpenvStorage/ovs/lib/alba.py", line 238, in add_osds
Jan 26 12:05:32 DEV-4N-199-191 celery[610]:     metadata=metadata)
Jan 26 12:05:32 DEV-4N-199-191 celery[610]:   File "/opt/OpenvStorage/ovs/lib/alba.py", line 389, in _add_generic_osds
Jan 26 12:05:32 DEV-4N-199-191 celery[610]:     for ip in osd.ips:
Jan 26 12:05:32 DEV-4N-199-191 celery[610]: TypeError: 'NoneType' object is not iterable
```

**Comment by simonOpenV** [comment]
Fixed in #555 
Packaging information: openvstorage-backend-1.12.0-dev.1517232557.9a80599

**Comment by simonOpenV** [comment]
Fixed in Master in #556 and #557
Packaging information: openvstorage-backend-1.12.0-dev.1517238185.a81ff78

**Comment by JeffreyDevloo** [comment]
Included in https://github.com/openvstorage/framework-alba-plugin/releases/tag/1.11.1 release
